Screenshot Framer CLI/ImagesParser.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  ImagesParser.swift
+//  Screenshot-Framer-CLI
+//
+//  Created by Patrick Kladek on 07.12.21.
+//  Copyright © 2021 Patrick Kladek. All rights reserved.
+//
+
+import Foundation
+
+final class ImagesParser {
+
+    struct Group {
+        struct Image {
+            let url: URL
+            var filename: String {
+                return self.url.lastPathComponent
+            }
+            var caption: String {
+                return self.url.deletingLastPathComponent().lastPathComponent
+            }
+        }
+        let images: [Image]
+        let name: String
+    }
+
+    struct Language {
+        let groups: [Group]
+        let language: String
+    }
+
+    struct Screen {
+        let groups: [Group]
+        let name: String
+    }
+
+    private let fileManager = FileManager()
+
+    // MARK: - ImagesParser
+
+    func languages(in folder: URL) throws -> [Language] {
```

Comment:
It is not fully obvious why the `ImagesParser` offers a method to read languages from a URL. Maybe a clarifying comment on top of the class helps here.